src/MicrosoftTeams.ts

```diff
@@ -475,6 +475,47 @@ namespace microsoftTeams
             authParams = authenticateParameters;
         }
 
+        /**
+         * Requests an AAD token to be issued on behalf of the app. The token is acquired token from the
+         * cache if it is not expired. Otherwise a request will be sent to AAD to obtain a new token.
+         * @param authTokenRequest A set pf values that configure the token request.
+         */
+        export function getAuthToken(authTokenRequest: AuthTokenRequest): void
+        {
+            ensureInitialized(frameContexts.content, frameContexts.settings, frameContexts.remove);
+
```

Comment:
So this isn't allowed in the auth context because we're assuming they're doing their own auth?